tools/schemapi/schemapi.py

```diff
@@ -539,7 +541,15 @@ def _passthrough(*args, **kwds):
             # Our class dict is constructed breadth-first from top to bottom,
             # so the first class that matches is the most general match.
             matches = self.class_dict[self.hash_schema(schema)]
-            cls = matches[0] if matches else _passthrough
+            if matches:
+                cls = matches[0]
+            elif default_class:
```

Comment:
maybe
```
else:
  cls = default_class or _passthrough
```

Reply:
My thinking was that we might want to check if `default_class` had been set or not, and I wasn't sure if something like `default_class == _passthrough` was safe.  But maybe we never need to check that anyway.

But if we can remove some of the cases then that sounds good to me.